R/ssb-utils.R

```diff
@@ -0,0 +1,47 @@
+#' extract_table
+#' @noRd
+extract_table <- function(url){
+  resp <- tryCatch({
+    GET(url, config(ssl_verifypeer = 0L))
+  }, error = function(e) {
+    message("Error in GET request: ", e)
+    return(NULL)
+  })
+
+  content <- httr::content(resp, "text")
+
+  data <- read.csv2(text = content, dec = ",")
```

Comment:
noAPI relies on the `{httr2}` package, and already has a set of built in function to handle queries and parsing of responses. We should preferably use these. See functions `send_query()` and `parse_response()` in the utils.R file.